prboom2/src/umapinfo.cpp

```diff
@@ -630,7 +630,6 @@ int ParseUMapInfo(const unsigned char *buffer, size_t length, umapinfo_errorfunc
 		if (parsed.endpic[0] && (strcmp(parsed.endpic, "-") != 0))
 		{
 			parsed.nextmap[0] = parsed.nextsecret[0] = 0;
-			if (parsed.endpic[0] == '!') parsed.endpic[0] = 0;
```

Comment:
Isn't the purpose of this line to handle `endgame = true`? If it is removed, how does that option work?